constd/main.go

```diff
@@ -104,84 +106,84 @@ func (c *constd) reconcileAtmo(errchan chan error) {
 }
 
 func (c *constd) reconcileConstellation(appSource appsource.AppSource, errchan chan error) {
-	runnables := appSource.Runnables()
+	apps := appSource.Applications()
 
-	for i := range runnables {
-		runnable := runnables[i]
+	for _, app := range apps {
+		runnables := appSource.Runnables(app.Identifier, app.AppVersion)
 
-		if _, exists := c.sats[runnable.FQFN]; !exists {
-			c.sats[runnable.FQFN] = newWatcher(runnable.FQFN, c.logger)
-		}
+		for i := range runnables {
+			runnable := runnables[i]
 
-		watcher := c.sats[runnable.FQFN]
+			c.logger.Info("reconciling", runnable.FQFN)
 
-		launch := func() {
-			cmd, port := satCommand(c.config, runnable)
+			if _, exists := c.sats[runnable.FQFN]; !exists {
+				c.sats[runnable.FQFN] = newWatcher(runnable.FQFN, c.logger)
+			}
 
-			// repeat forever in case the command does error out
-			uuid, pid, err := exec.Run(
-				cmd,
-				"SAT_HTTP_PORT="+port,
-				"SAT_CONTROL_PLANE="+c.config.controlPlane,
-				"SAT_ENV_TOKEN="+c.config.envToken,
-			)
+			watcher := c.sats[runnable.FQFN]
```

Comment:
is changing the name of this variable in scope for this PR? It's shadowing the name of the struct

Reply:
Good call, fixed.